rfc/003-skip-values.md

```diff
@@ -0,0 +1,157 @@
+# RFC: Skip Values
+
+- **Status**: Draft
+- **Author(s)**: @ericbuckley
+- **Creation Date**: 2025-02-13
+- **Last Updated**: 2025-02-14
+- **RFC ID**: RFC-003
+
+---
+
+## Summary
+
+There are instances where placeholder values are used in fields to indicate missing data.
+In such cases, comparing two placeholder fields is inaccurate, as the presence of missing
+data in both documents does not imply they belong to the same individual. This RFC
+proposes a system that allows users to specify conditions under which these values should
+be excluded from comparisons.
+
+## Motivation
+
+Healthcare organizations often encounter incomplete or missing data and commonly use
+placeholder values to indicate this, such as "Unknown" or "Not Specified." In some cases,
+placeholders are field-specific, like using "John Doe" or "Jane Doe" for missing patient
+names. Comparing two records with the placeholder name "John Doe" does not imply they
+belong to the same individual, as the value is a stand-in rather than an actual data point.
```

Comment:
You might note it later on, but there's a related issue too: comparing a John Doe to John Smith shouldn't be taken as first name agreement, either, which means we need a skip value system that's aware across fields (in an ideal world)

Reply:
The new feature, `NAME` as noted below, is intended to address this.